spec/features/company_view_spec.rb

```diff
@@ -0,0 +1,136 @@
+require 'rails_helper'
+
+describe "the company view", type: :feature do
+  let(:company) { Company.create(name: "Apple") }
+
+  describe "#phone_numbers" do
+
+    before(:each) do
+      company.phone_numbers.create(number: "867-5309")
+      company.phone_numbers.create(number: "555-1234")
+
+      visit company_path(company)
+    end
+
+    it "shows the phone numbers" do
+      company.phone_numbers.each do |phone|
+        expect(page).to have_content(phone.number)
+      end
+    end
+
+    it "has a link to add a new phone number" do
+      expect(page).to have_link('Add phone number', href: new_phone_number_path(contact_id: company.id, contact_type: 'Company'))
```

Comment:
Line is too long. [129/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.